foundations/installations/prerequisites.md

```diff
@@ -257,7 +257,7 @@ Currently there is a bug preventing CloudReady v83.4 from successfully installin
 #### Why Google Chrome?
 
 Because our lessons use Google Chrome, and Chrome/Chromium are overwhelmingly used by developers and consumers, the recommendations we make are very intentional.
-Look at this [breakdown of browser market shares](https://gs.statcounter.com/browser-market-share#monthly-202110-202110-bar) and see what other folks are using the most
+Look at this [usage share of web browsers](https://en.wikipedia.org/wiki/Usage_share_of_web_browsers) and see what other folks are using the most
```

Comment:
I think we should link to the exact section where the usage share is listed (the Summary Tables section). Additionally, would you mind adding a period at the end of this sentence?

Reply:
Done.  New [commit](https://github.com/TheOdinProject/curriculum/pull/23897/commits/6619bfdd9f7e1297583aa6450ce31e15d785931d)